doc/rest.md

```diff
@@ -21,6 +21,9 @@
 #### REST path parameters
 - **workspace_name**, string `^[a-z][a-z0-9]*(_[a-z0-9]+)*$`
    - string identifying [workspace](models.md#workspace)
+  
+
+**_NOTE:_** Before version 1.10.0, endpoints did not include `/workspaces/` in their path. Such endpoints are now redirected to the new one and will return header **Deprecated**. Please, if you get such header in response, rewrite your client to use new endpoint path. Old endpoint paths will stop responding after the next major release.
```

Comment:
**_NOTE:_** Before version 1.10.0, workspace-related endpoints did not include `/workspaces` in their path. These old endpoints are still functional, but deprecated. More specifically, they return HTTP header **Deprecated**. If you get such header in response, rewrite your client to use new endpoint path. Old endpoints will stop working in the next major release.